ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js

```diff
@@ -40,7 +40,13 @@ var pixelOffset = new Cesium.Cartesian2(0.0, 0);
 Cesium.BingMapsApi.defaultKey = properties.bingKey || 0;
 var imageryProviderTypes = CesiumLayerCollectionController.imageryProviderTypes;
 
-function setupTerrainProvider(viewer, { type, ...terrainConfig } = {}) {
+function setupTerrainProvider(viewer, terrainProvider ) {
+    if (terrainProvider == null || terrainProvider === undefined) {
```

Comment:
❓ Should this be `if(terrainProvider === null || terrainProvider === undefined)`?

`null == undefined` evaluates to `true` so the second conditional will never get run as is.